tests/actions.go

```diff
@@ -811,3 +868,470 @@ func checkoutTag(tagName string) error {
 
 	return nil
 }
+
+//scheduler backup, make sure there are serveral backup record is generated, and 
```

Comment:
Please remove all the backup related codes from this PR. It should be another PR.